database/factories/ModelFactory.php

```diff
@@ -22,15 +20,30 @@
  * @see https://laravel.com/docs/5.5/database-testing#writing-factories
  */
 
+// Action Factory
```

Comment:
I moved this to the top of the file instead of the bottom, to give more context to setting the `school_id` in the Post factory  below.